crawl-ref/source/tileweb.cc

```diff
@@ -982,6 +982,21 @@ template<class T> static bool _update_int(bool force, T& current, T next,
         return false;
 }
 
+static bool _update_bool(bool force, bool& current, bool next,
```

Comment:
because `_update_int` is templated, it works fine with `current`/`next` of type `bool`. It does send 0/1 in the json, but this is ok from the client's perspective. So I'm not sure adding a new function is necessary